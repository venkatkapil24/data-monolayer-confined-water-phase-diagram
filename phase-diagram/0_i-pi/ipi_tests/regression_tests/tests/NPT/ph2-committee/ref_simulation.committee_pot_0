 #EXTRAS(committee_pot)# Step:           0  Bead:      0  
    -0.04207446          -0.04207446          -0.04207446
 #EXTRAS(committee_pot)# Step:           1  Bead:      0  
    -0.04207940          -0.04207940          -0.04207940
 #EXTRAS(committee_pot)# Step:           2  Bead:      0  
    -0.04208532          -0.04208532          -0.04208532
 #EXTRAS(committee_pot)# Step:           3  Bead:      0  
    -0.04209157          -0.04209157          -0.04209157
 #EXTRAS(committee_pot)# Step:           4  Bead:      0  
    -0.04209860          -0.04209860          -0.04209860
 #EXTRAS(committee_pot)# Step:           5  Bead:      0  
    -0.04210648          -0.04210648          -0.04210648
 #EXTRAS(committee_pot)# Step:           6  Bead:      0  
    -0.04211590          -0.04211590          -0.04211590
 #EXTRAS(committee_pot)# Step:           7  Bead:      0  
    -0.04212662          -0.04212662          -0.04212662
 #EXTRAS(committee_pot)# Step:           8  Bead:      0  
    -0.04213783          -0.04213783          -0.04213783
 #EXTRAS(committee_pot)# Step:           9  Bead:      0  
    -0.04215014          -0.04215014          -0.04215014
 #EXTRAS(committee_pot)# Step:          10  Bead:      0  
    -0.04216357          -0.04216357          -0.04216357
