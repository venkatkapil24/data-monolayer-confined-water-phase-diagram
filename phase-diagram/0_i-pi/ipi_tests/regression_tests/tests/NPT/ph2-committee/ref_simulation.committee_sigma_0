 #EXTRAS(committee_uncertainty)# Step:           0  Bead:      0  
0.0
 #EXTRAS(committee_uncertainty)# Step:           1  Bead:      0  
0.0
 #EXTRAS(committee_uncertainty)# Step:           2  Bead:      0  
0.0
 #EXTRAS(committee_uncertainty)# Step:           3  Bead:      0  
8.498374721940739e-18
 #EXTRAS(committee_uncertainty)# Step:           4  Bead:      0  
0.0
 #EXTRAS(committee_uncertainty)# Step:           5  Bead:      0  
0.0
 #EXTRAS(committee_uncertainty)# Step:           6  Bead:      0  
0.0
 #EXTRAS(committee_uncertainty)# Step:           7  Bead:      0  
0.0
 #EXTRAS(committee_uncertainty)# Step:           8  Bead:      0  
0.0
 #EXTRAS(committee_uncertainty)# Step:           9  Bead:      0  
0.0
 #EXTRAS(committee_uncertainty)# Step:          10  Bead:      0  
8.498374721940739e-18
