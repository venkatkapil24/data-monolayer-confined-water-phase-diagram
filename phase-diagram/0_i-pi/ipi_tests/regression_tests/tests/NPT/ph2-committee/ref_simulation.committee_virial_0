 #EXTRAS(committee_virial)# Step:           0  Bead:      0  
    -0.07228983           0.00002614          -0.00001933           0.00000000          -0.07218501           0.00005912           0.00000000           0.00000000          -0.07223799
    -0.07228983           0.00002614          -0.00001933           0.00000000          -0.07218501           0.00005912           0.00000000           0.00000000          -0.07223799
    -0.07228983           0.00002614          -0.00001933           0.00000000          -0.07218501           0.00005912           0.00000000           0.00000000          -0.07223799
 #EXTRAS(committee_virial)# Step:           1  Bead:      0  
    -0.07227461           0.00003090          -0.00001629           0.00000000          -0.07216252           0.00006169           0.00000000           0.00000000          -0.07221225
    -0.07227461           0.00003090          -0.00001629           0.00000000          -0.07216252           0.00006169           0.00000000           0.00000000          -0.07221225
    -0.07227461           0.00003090          -0.00001629           0.00000000          -0.07216252           0.00006169           0.00000000           0.00000000          -0.07221225
 #EXTRAS(committee_virial)# Step:           2  Bead:      0  
    -0.07225812           0.00003581          -0.00001321           0.00000000          -0.07213839           0.00006477           0.00000000           0.00000000          -0.07218471
    -0.07225812           0.00003581          -0.00001321           0.00000000          -0.07213839           0.00006477           0.00000000           0.00000000          -0.07218471
    -0.07225812           0.00003581          -0.00001321           0.00000000          -0.07213839           0.00006477           0.00000000           0.00000000          -0.07218471
 #EXTRAS(committee_virial)# Step:           3  Bead:      0  
    -0.07223981           0.00004080          -0.00000989           0.00000000          -0.07211182           0.00006800           0.00000000           0.00000000          -0.07215491
    -0.07223981           0.00004080          -0.00000989           0.00000000          -0.07211182           0.00006800           0.00000000           0.00000000          -0.07215491
    -0.07223981           0.00004080          -0.00000989           0.00000000          -0.07211182           0.00006800           0.00000000           0.00000000          -0.07215491
 #EXTRAS(committee_virial)# Step:           4  Bead:      0  
    -0.07221989           0.00004555          -0.00000628           0.00000000          -0.07208332           0.00007156           0.00000000           0.00000000          -0.07212295
    -0.07221989           0.00004555          -0.00000628           0.00000000          -0.07208332           0.00007156           0.00000000           0.00000000          -0.07212295
    -0.07221989           0.00004555          -0.00000628           0.00000000          -0.07208332           0.00007156           0.00000000           0.00000000          -0.07212295
 #EXTRAS(committee_virial)# Step:           5  Bead:      0  
    -0.07219863           0.00005019          -0.00000262           0.00000000          -0.07205304           0.00007541           0.00000000           0.00000000          -0.07208897
    -0.07219863           0.00005019          -0.00000262           0.00000000          -0.07205304           0.00007541           0.00000000           0.00000000          -0.07208897
    -0.07219863           0.00005019          -0.00000262           0.00000000          -0.07205304           0.00007541           0.00000000           0.00000000          -0.07208897
 #EXTRAS(committee_virial)# Step:           6  Bead:      0  
    -0.07217642           0.00005493           0.00000117           0.00000000          -0.07202088           0.00007936           0.00000000           0.00000000          -0.07205322
    -0.07217642           0.00005493           0.00000117           0.00000000          -0.07202088           0.00007936           0.00000000           0.00000000          -0.07205322
    -0.07217642           0.00005493           0.00000117           0.00000000          -0.07202088           0.00007936           0.00000000           0.00000000          -0.07205322
 #EXTRAS(committee_virial)# Step:           7  Bead:      0  
    -0.07215263           0.00005995           0.00000521           0.00000000          -0.07198718           0.00008349           0.00000000           0.00000000          -0.07201608
    -0.07215263           0.00005995           0.00000521           0.00000000          -0.07198718           0.00008349           0.00000000           0.00000000          -0.07201608
    -0.07215263           0.00005995           0.00000521           0.00000000          -0.07198718           0.00008349           0.00000000           0.00000000          -0.07201608
 #EXTRAS(committee_virial)# Step:           8  Bead:      0  
    -0.07212677           0.00006490           0.00000935           0.00000000          -0.07195091           0.00008780           0.00000000           0.00000000          -0.07197669
    -0.07212677           0.00006490           0.00000935           0.00000000          -0.07195091           0.00008780           0.00000000           0.00000000          -0.07197669
    -0.07212677           0.00006490           0.00000935           0.00000000          -0.07195091           0.00008780           0.00000000           0.00000000          -0.07197669
 #EXTRAS(committee_virial)# Step:           9  Bead:      0  
    -0.07209935           0.00006967           0.00001363           0.00000000          -0.07191278           0.00009218           0.00000000           0.00000000          -0.07193499
    -0.07209935           0.00006967           0.00001363           0.00000000          -0.07191278           0.00009218           0.00000000           0.00000000          -0.07193499
    -0.07209935           0.00006967           0.00001363           0.00000000          -0.07191278           0.00009218           0.00000000           0.00000000          -0.07193499
 #EXTRAS(committee_virial)# Step:          10  Bead:      0  
    -0.07207068           0.00007466           0.00001834           0.00000000          -0.07187325           0.00009686           0.00000000           0.00000000          -0.07189178
    -0.07207068           0.00007466           0.00001834           0.00000000          -0.07187325           0.00009686           0.00000000           0.00000000          -0.07189178
    -0.07207068           0.00007466           0.00001834           0.00000000          -0.07187325           0.00009686           0.00000000           0.00000000          -0.07189178
