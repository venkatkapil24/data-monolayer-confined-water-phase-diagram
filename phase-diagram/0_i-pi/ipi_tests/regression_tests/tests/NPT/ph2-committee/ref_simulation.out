# column   1     --> step : The current simulation time step.
# column   2     --> time{picosecond} : The elapsed simulation time.
# column   3     --> conserved : The value of the conserved energy quantity per bead.
# column   4     --> temperature{kelvin} : The current temperature, as obtained from the MD kinetic energy.
# column   5     --> potential : The physical system potential energy.
# column   6     --> volume : The volume of the cell box.
    0.00000000e+00     0.00000000e+00    -1.45005809e-02     3.01137246e+01    -4.20744571e-02     3.83612063e+04   
    1.00000000e+00     1.00000000e-03    -1.45005814e-02     3.00518573e+01    -4.20794037e-02     3.83611477e+04   
    2.00000000e+00     2.00000000e-03    -1.45005818e-02     3.01399779e+01    -4.20853190e-02     3.83607996e+04   
    3.00000000e+00     3.00000000e-03    -1.45005824e-02     3.00146042e+01    -4.20915713e-02     3.83604865e+04   
    4.00000000e+00     4.00000000e-03    -1.45005830e-02     2.99832963e+01    -4.20985958e-02     3.83600131e+04   
    5.00000000e+00     5.00000000e-03    -1.45005837e-02     2.99936885e+01    -4.21064789e-02     3.83593054e+04   
    6.00000000e+00     6.00000000e-03    -1.45005844e-02     3.00833066e+01    -4.21159033e-02     3.83580327e+04   
    7.00000000e+00     7.00000000e-03    -1.45005852e-02     3.01364798e+01    -4.21266206e-02     3.83563030e+04   
    8.00000000e+00     8.00000000e-03    -1.45005860e-02     3.01656799e+01    -4.21378344e-02     3.83545134e+04   
    9.00000000e+00     9.00000000e-03    -1.45005869e-02     3.03146694e+01    -4.21501387e-02     3.83523528e+04   
    1.00000000e+01     1.00000000e-02    -1.45005879e-02     3.04031991e+01    -4.21635743e-02     3.83497492e+04   
